src/collections/maple_tree/mod.rs

```diff
@@ -1052,46 +1212,156 @@ mod tests {
         }
     }
 
+    #[test]
+    fn test_range_node_condense_node() {
+        let mut rn: RangeLeaf<usize, usize> = RangeLeaf::new();
+        let min = 0;
+        let max = std::usize::MAX;
```

Comment:
Small detail, just use std::usize::MAX, and use MAX instead of max in the codebase because it's a constant :)

Reply:
ahh true haha didn't think about that